java/src/jmri/jmrix/cmri/serial/serialmon/Bundle.java

```diff
@@ -24,7 +24,7 @@
 public class Bundle extends jmri.jmrix.cmri.serial.Bundle {
 
     @Nullable
-    private static final String name = "jmri.jmrix.cmri.serial.serialmon.SerialFilterBundle"; // NOI18N
+    private static final String name = "jmri.jmrix.cmri.serial.serialmon.SerialFilterBundle"; // c2
```

Comment:
Please retain the NOI18N comments unless you want someone to localize the String. See http://jmri.org/help/en/html/doc/Technical/I8N.shtml for an explanation.

Reply:
Changed and read doc.